controllers/chat.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const mongo = require('../models/chat');
+const router = express.Router();
+
+let name;
```

Comment:
Don't define these variables outside. Concurrency issue hoga, "deadlock" in OS jaisa.